### PR TITLE
Test/klbp 386 update titre de sejour docs

### PR DESCRIPTION
**Pull request by mariapitechplus**

### CODE REVIEW

File: src/__tests__/pages/steps/Step3PrivateCompany.test.tsx [hunk 1]
```diff
@@ -377,17 +378,18 @@ test('Choosing "Oui" to the radio button, cards section is displayed - WITH seco
...
     ).toBeInTheDocument()
-    expect(screen.getByText('Document validé')).toBeInTheDocument()
+    expect(screen.getAllByText('Document validé')).toHaveLength(2)
 
     expect(
       screen.queryByText('Ajouter un nouveau Bénéficiaire Effectif')
     ).toBeInTheDocument()
   })
-})
+}, 10000)
```
Comment (mariuspop86) [inline src/__tests__/pages/steps/Step3PrivateCompany.test.tsx:392]: Why do we need this timeout

**Review by ioan-iacob-netea — APPROVED** [review]

File: src/__tests__/pages/steps/Step3PrivateCompany.test.tsx (hunk 1 above)
Comment (mariapitechplus) [reply src/__tests__/pages/steps/Step3PrivateCompany.test.tsx]: We don't :D

Comment (mariapitechplus) [reply src/__tests__/pages/steps/Step3PrivateCompany.test.tsx]: fixed

File: src/enums/step2Enums.ts [hunk 2]
```diff
@@ -61,11 +66,11 @@ export const getInitialFilesState = (
   }
 
   switch (identityDocumentType) {
```
Comment (mariuspop86) [inline src/enums/step2Enums.ts:68]: Do we really need a switch here? I think an if is fine.

**Review by mariuspop86 — APPROVED** [review]

File: src/enums/step2Enums.ts (hunk 2 above)
Comment (mariapitechplus) [reply src/enums/step2Enums.ts]: refactored

**Pull request merged**